src/elements/content-sidebar/activity-feed/task-new/messages.js

```diff
@@ -89,6 +94,11 @@ const messages = defineMessages({
         defaultMessage: 'IN PROGRESS',
         description: 'Label for a task in progress (in upper-case in supported language)',
     },
+    taskFeedHeadlinePreviewCurrentUser: {
+        id: 'be.contentSidebar.activityFeed.taskNew.tasksFeedHeadlinePreviewCurrentUser',
+        defaultMessage: '{user}',
```

Comment:
Is this message needed? Can the value be rendered directly?

Reply:
I believe this message is needed as the values for this message are rendered by component `UserLink`

Reply:
It looks like the value of `user` is the UserLink component. Can we render the UserLink component directly?